sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigtable/BigtableConfig.java

```diff
@@ -151,47 +151,36 @@ BigtableConfig withValidate(boolean isEnabled) {
 
   @VisibleForTesting
   BigtableConfig withBigtableService(BigtableService bigtableService) {
-    checkNotNull(bigtableService, "bigtableService can not be null");
+    checkArgument(bigtableService != null, "bigtableService can not be null");
     return toBuilder().setBigtableService(bigtableService).build();
   }
 
   void validate() {
-    checkArgument(getProjectId() != null && !getProjectId().isEmpty()
+    checkArgument(getTableId() != null, "Could not obtain Bigtable table id");
```

Comment:
If the table id is accessible, check that it isn't empty.

Reply:
Done